index.php

```diff
@@ -21,10 +21,10 @@
 		<meta http-equiv="content-language" content="<? echo $lang; ?>" />
 		<meta name="keywords" content="openstreetmap, openrailwaymap, alexander matheisen, rurseekatze, openlayers, osm, matheisen, orm, eisenbahnkarte, bahnkarte, railmap, railway, railways, eisenbahn, streckenkarte" />
 		<meta name="title" content="OpenRailwayMap" />
-		<meta name="author" content="rurseekatze, Alexander Matheisen" />
+		<meta name="author" content="Unknown" />
```

Comment:
Why do you replace the current value by "Unknown"? You can add yourself to this tag if you like, but "Unknown" does not make sense.